### PR TITLE
CASINT-3844: microTip documentation

### PR DESCRIPTION
**Pull request by remenska**

### CODE REVIEW

**Review by homiak — DISMISSED** [review]

File: src/source/ads.rst [hunk 1]
```diff
@@ -395,6 +396,17 @@ per customer. Can either be set when creating an ad or when updating an
 existing ad. However, once set, it can no longer be modified. When fetching an
 existing ad which does not have a **vendorId**, the field is omitted.
 
+.. index:: microTip
+.. _ad_microTip:
+
+microTip
+""""""""
+A short freeform text with a maximum length of 18 characters, excluding any characters in ``.,/@#<>``.
+It is a feature used in the context of SMB bundles (currently only by Marktplaats tenant),
```
Comment (twessling-icas) [inline src/source/ads.rst]: our sellside customers don't know the concept of SMB or the word 'bundles'; Please reword using information about packages & features.

File: src/source/ads.rst [hunk 2]
```diff
@@ -395,6 +396,17 @@ per customer. Can either be set when creating an ad or when updating an
...
+and will only be usable by enabled sellers.
+It will be shown as a "sticker" (giving extra attention) on the ad in the search results.
```
Comment (twessling-icas) [inline src/source/ads.rst]: remove the 'sticker' mention, it's there to give extra attention.

**Review by twessling-icas — CHANGES_REQUESTED** [review]

File: src/source/ads.rst (hunk 1 above)
Comment (remenska) [reply src/source/ads.rst]: hm, I thought "bundles" was even in the link of the packaging..do you happen to have that link, I lost it together with the lingo we are using in it

Comment (remenska) [reply src/source/ads.rst]: found it, nevermind

File: src/source/ads.rst (hunk 2 above)
Comment (remenska) [reply src/source/ads.rst]: fixed

File: src/source/ads.rst (hunk 1 above)
Comment (remenska) [reply src/source/ads.rst]: fixed

**Review by Bojhan — APPROVED** [review]
We can add this to our documentation.

**Pull request merged**